### ISSUE TITLE
Potions are stacked

### ISSUE THREAD

**Issue by Momonche** [spigot-gillesm/RPGChest-Premium#12]
Hello Gilles !

When I try to put more than one potion in a RPGChest, they are stacked but in Minecraft, potions are unstacked

```
RP_8:
  Min: 4
  Max: 4
  Items:
    POTION:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 3
      Max: 3
    EMERALD:
      Material: EMERALD
      Chance: 1
      Min: 1
      Max: 1
```
![mZZlZXD](https://user-images.githubusercontent.com/37519498/72827757-aa0f6100-3c7b-11ea-9cda-ac8d8d7d7a2e.png)

I try this but only one potion appear in the chest : 

```
RP_8:
  Min: 4
  Max: 4
  Items:
    POTION1:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 1
      Max: 1
    POTION2:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 1
      Max: 1
    POTION3:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 1
      Max: 1
    EMERALD:
      Material: EMERALD
      Chance: 1
      Min: 1
      Max: 1
```
![YQEvsXk](https://user-images.githubusercontent.com/37519498/72827855-d75c0f00-3c7b-11ea-921f-271913ac4d3d.png)

**Comment by spigot-gillesm** [comment]
I can unstack them if you prefer ;) Or even make an option to let users choose if they want it stacked or not! :D

**Comment by Momonche** [comment]
An option in the potion config would be a great idea, Users will be able to choose for themselves !

I think about something like this : 
```
    POTION1:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Stack: true/false
      Chance: 1
      Min: 1
      Max: 1
```

**Comment by spigot-gillesm** [comment]
Yup! I'll do that ;) Thanks for the suggestion!

**Comment by spigot-gillesm** [comment]
Done! Everything should be configurable now ;)

**Comment by Momonche** [comment]
Can you update the wiki please ?
I don't know how to do exactly and I maybe have a problem with that

**Comment by spigot-gillesm** [comment]
I added the option in the Item section page of the Item table section ;) It's just a simple option that spawns in the menu when you create any kind of potion. The icon is a chest

**Comment by Momonche** [comment]
I don't understand what I'm doing wrong with this table : 

```
RP_8:
  Min: 4
  Max: 4
  Items:
    POTION:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 3
      Max: 3
      Stack: false
    EMERALD:
      Material: EMERALD
      Chance: 1
      Min: 1
      Max: 1
```

With this table I must have 3 Potions and 1 Emerald EVERY TIME ?
But I have this : 
![JpJ0Aai](https://user-images.githubusercontent.com/37519498/72931327-efa45a80-3d5d-11ea-947c-695914fc4ac1.png)

No error in console

**Comment by Momonche** [comment]
I just realise all items in all chests are 2, something is broken visibly

**Comment by spigot-gillesm** [comment]
No, everything you did is right, there's something wrong on my end. I'll fix it as fast as I can

**Comment by spigot-gillesm** [comment]
Fixed! (I hope)

**Comment by Momonche** [comment]
It's fix with all items but not fixed with potion, there is always two potion when stack is on false (but it's ok when stack is on true)  : 


A test with 3 potions and 1 emerald
```
test:
  Min: 4
  Max: 4
  Items:
    POTION:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 3
      Max: 3
      Stack: false
    EMERALD:
      Material: EMERALD
      Chance: 1
      Min: 1
      Max: 1
```
![626l9Bb](https://user-images.githubusercontent.com/37519498/72999604-e370d980-3dff-11ea-9bcc-214363c05b88.png)


A test with 4 potions and 1 emerald
```
test:
  Min: 5
  Max: 5
  Items:
    POTION:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 4
      Max: 4
      Stack: false
    EMERALD:
      Material: EMERALD
      Chance: 1
      Min: 1
      Max: 1
```
![WqkubLt](https://user-images.githubusercontent.com/37519498/72999674-faafc700-3dff-11ea-863c-5cea5f2aaa69.png)



A test with 1 potion and 1 emerald
```
test:
  Min: 2
  Max: 2
  Items:
    POTION:
      Material: POTION
      Effect: SPEED
      Upgraded: true
      Extended: false
      Chance: 1
      Min: 1
      Max: 1
      Stack: false
    EMERALD:
      Material: EMERALD
      Chance: 1
      Min: 1
      Max: 1
```
![TQcq0PY](https://user-images.githubusercontent.com/37519498/72999840-3ba7db80-3e00-11ea-8e16-3b88ba423273.png)

**Comment by spigot-gillesm** [comment]
Oh crap, sorry about that. It will be fixed tonight or tomorrow! Sorry for the inconvenience

**Comment by Momonche** [comment]
Fixed on 1.9.3.2 !

Thanks !
